buildlib/azure-pipelines.yml

```diff
@@ -28,7 +28,7 @@ stages:
 
           - bash: |
               set -eE
-              ./contrib/configure-release --with-java
+              ./contrib/configure-release --with-java --enable-logging
```

Comment:
does it affect libjucx.so only ?

Reply:
Yes, because jucx can be run on a dev builds, with lower logging level.

Reply:
in this case, log level is determined by the ucx library, it should be independent of how jucx is built

Reply:
```
#define ucs_log_component_is_enabled(_level, _comp_log_config) \
    ucs_unlikely(((_level) <= UCS_MAX_LOG_LEVEL) && \
                 ((_level) <= (((ucs_log_component_config_t*)(_comp_log_config))->log_level)))
```

When we compile jucx with `--disable-logging` then the compiler may simple remove all calls to the `ucs_trace`, `ucs_debug` from jucx. Because at compile time `UCS_MAX_LOG_LEVEL` is known

Reply:
i see, jucx is also calling ucs_XX ,macros..
but wouldn't it affect jucx performance, if we always enable logging for it?
maybe set UCS_MAX_LOG_LEVEL to DEBUG by default, just for jucx somehow?

Reply:
But there's a second condition in macro: if requested log_level < configured log level. So it won't do logging if you run with release version of ucx that has default log_level = warn. Just 1 condition check.

Reply:
can we modify the configured log level just for jucx?
logging code, even if it's just compiled-in and not run, has some performance overhead. do we have any logging in fast path?
alternatively, can we have a separate debug build of jucx?